spec/swagger.yaml

```diff
@@ -144,6 +144,11 @@ parameters:
     in: query
     type: string
     description: The partial search of the text fields.
+  collectionCriteria:
+    name: criteria
```

Comment:
Please, ensure that `collectionCriteria` is specified for:
Transaction report, Cancellation report, Cumulative subscriptions and cumulative plan subscriptions, retention value, .